conf/messages

```diff
@@ -197,6 +197,27 @@ help.cookies.how_used.satisfaction_survey.table.ep202.name=ep202
 help.cookies.how_used.satisfaction_survey.table.ep202.purpose=This works with ep201 to help us identify how you use the survey
 help.cookies.how_used.satisfaction_survey.table.ep202.expires=1 year
 
+help.cookies.how_used.digital_assistant.heading=Digital assistant cookies
+help.cookies.how_used.digital_assistant.info=The HMRC digital assistant and webchat can be found on various HMRC services pages. For the digital assistant and website to work correctly, the following cookies are required:
```

Comment:
I can see this is the supplied content, but I wonder if this was meant to be "For the digital assistant and **webchat** to work correctly" ?